src/styles/blocks/card.scss

```diff
@@ -0,0 +1,108 @@
+$cardWidth: 200px;
+$buttonHeight: 40px;
+$cardLargeTextColor: #060b35;
+$cardSmallTextColor: #616070;
+$cardBorderColor: #f3f3f3;
+$buttonColor: #00acdc;
```

Comment:
You are defining the same color variables in multiple files - a better solution is having a dedicated `variables.scss` or `common.scss` and put your variables there.